_layouts/showcase-item.html

```diff
@@ -183,16 +195,15 @@ <h3 style="opacity: 0.75; margin-top: 0.5rem">
 			<div class="card youtube" style="padding-bottom: 56.25%">
 				<iframe src="https://www.youtube-nocookie.com/embed/{{ page.youtube_id }}"
 					allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-					allowfullscreen="" frameborder="0">
+					allowfullscreen="">
 				</iframe>
 			</div>
 			{% endif %}
 
 			{% for image in page.gallery %}
 			<a href="{{ image }}" class="lightbox" data-group="showcase">
 				<img class="rounded-lg" src="{{ image }}" alt="Screenshot of {{ page.title }} by {{ page.author }}"
-					style="width: 100%; height: auto; margin-top: 1rem; background:  linear-gradient(90deg, #947451 12%, #a8885f 29%, #8c8977 65%, #d2ba9f 98%) "
-					width="16" height="9">
+					width="16" height="9" onload="removeFeatureImgBackground(this)">
```

Comment:
Instead of doing this with JavaScript I would simply add 
`style="background: unset; background-color: unset; border: unset;"` to that element instead of the `onload`

Reply:
Did a test myself and with just `style="background: transparent;"` it does the trick

Reply:
If you do that also with the home page change we should be good to merge! thanks (:

Reply:
Isn't the background used as a placeholder to show there is an image loading?
![image-loading](https://github.com/godotengine/godot-website/assets/48391407/7046b30c-2525-4700-b279-69117aa466ce)

If we remove the background then there is no indication that there is supposed to be an image here before the image actually appearing
![image-loading-without-background](https://github.com/godotengine/godot-website/assets/48391407/27a16aa1-61cf-4f15-b02f-dec0e42fe085)

Reply:
True! missed that since it was loading so fast locally... I'm not sure is super necessary to make the progressive loading happen, specially now that the images are webp and load faster 😅

Reply:
WebP can still load slowly if on a slow mobile network (3G/4G with bad coverage), so placeholders are still important.

Reply:
I think it makes sense to leave the placeholder, even tho is faster now it still improves the experience for some users.

I kinda sidetracked a bit and ended up optimizing other stuff unrelated to images, I promise I won't change anything else. Loading times are much faster now and touching 100 performance score :D

I found a gem called jekyll_picture_tag that helped a lot, I've used it where it made sense.